### ISSUE TITLE
[FEATURE] support files with no extension on bunkr

### ISSUE THREAD

**Issue by baccccccc** [jbsparrow/CyberDropDownloader#200]
Here are a few perfectly working links. However, the files are uploaded without an extension. Which is stupid IMO but you do you.

```
https://bunkrrr.org/d/MissAlexaPearl---Fucking-Step-Bro-After-School-tsYbtd7G
https://bunkrrr.org/d/MissAlexaPearl---Step-Bro-Gets-A-Taste-a18Xwnvp
https://bunkrrr.org/d/MissAlexaPearl---Squirting-Her-Milk-On-You-XE3vAnGp
```

today all such files a legitimately skipped by CDL during scrape, with `No File Extension` message.

can we instead download them as is?

**Comment by jbsparrow** [comment]
Can probably implement a config option to download files that have no extension. No clue why someone would upload them without an extension, maybe to try to avoid a DMCA takedown?

**Comment by baccccccc** [comment]
> No clue why someone would upload them without an extension, maybe to try to avoid a DMCA takedown?

I have no idea either, maybe it's a human error. Either way, I feel like downloading these files would be still more helpful than reporting as an error. Thanks for looking into this!

**Comment by jbsparrow** [comment]
I looked into ways of detecting the file type using its headers. Unfortunately that doesn't look like something that I can implement, as the best package I found for the job requires a library that is not available on PyPi for all platforms. It requires the user to manually use `apt-get` if they are on linux, `pip` if they are on windows, and `brew` if they are on MacOS. 

Unfortunately it looks like I'll either just make a config option to allow downloading files with no extension, or just leave it as it is.

How often do you find a file with no extension that you are trying to download? I feel like it wouldn't be very often?

**Comment by baccccccc** [comment]
yeah, not very often. I would appreciate the config option, though. Thanks!

**Comment by Ronar000** [comment]
Ever since Bunkr updated their UI, I can never mass download using the app. Any fix coming soon?

**Comment by NTFSvolume** [comment]
> Ever since Bunkr updated their UI, I can never mass download using the app. Any fix coming soon?

This was fixed the day after they updated the UI. Check which version are you running and update. Latest version is v5.7.1
